src/shadowbox/server/manager_service.ts

```diff
@@ -60,6 +60,11 @@ interface ResponseType {
   send(code: number, data?: {}): void;
 }
 
+enum Success {
```

Comment:
Consider naming `HttpSuccess`, `RestSuccess`, or `SuccessCodes` to further qualify.

Reply:
Done.